src/crackdefault

```diff
@@ -1,37 +1,42 @@
-#!/system/bin/sh
 # This script is part of airbash (https://github.com/tehw0lf/airbash)
 # Written by tehw0lf
 # Execution of this script will search the handshake database for entries
 # that have not been cracked yet and then try to match the SSID with
 # routers that have known vulnerabilities. Subsequently, the default
 # passwords for any found matches are calculated and tested using aircrack-ng (https://aircrack-ng.org)
 
+
+AIRCRACK_BIN=`which aircrack-ng`
+SQLITE3_BIN=`which sqlite3`
+
 # configure path and file names
 export PATH="$PATH:modules"
-path=/path/to/airbash-folder/
-hs=.hs/
-wl=.wl/
-db=.db.sqlite3
+WDIR=`pwd`  # android
+#WDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"  # linux
```

Comment:
As above - prevents execution on POSIX shells, but should be uncommented when using Bash.